src/movement/rotation_engine.cpp

```diff
@@ -7,6 +7,29 @@ phi = pitch
 theta = yaw
```

Comment:
Will add comments. There's a few things I want to work on but this control system rework got out of hand as it is.